scripts/get_probs.py

```diff
@@ -56,7 +56,7 @@ def main(args):
             print("loading calibration data")
             np.random.shuffle(data)
             cal_data = data[:n_calib]
-            X_cal, y_cal = get_sims_labels(cal_data, partial=False)
+            X_cal, y_cal = get_sims_labels(cal_data, partial=True)
```

Comment:
@ronboger this should be True since this is the partial computation, right? will change if not